cli/src/commands/extensions/api.rs

```diff
@@ -187,13 +187,14 @@ async fn get_package_details(
 /// Equivalent to `phylum parse`.
 #[op]
 async fn parse_lockfile(
-    op_state: Rc<RefCell<OpState>>,
+    _op_state: Rc<RefCell<OpState>>,
     lockfile: String,
     lockfile_type: Option<String>,
 ) -> Result<PackageLock> {
+    // TODO: Validate read permissions
```

Comment:
This seems to be the biggest question left to answer. What will this permission check look like?